{"name": "test_login_propio", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'driver'", "trace": "setup_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0fe3a5481da884b2695eb161ea3cbf6e\")>\n\n    @allure.feature('Login Tests')\n    def test_login_propio(setup_browser):\n        driver = setup_browser\n    \n        try:\n    \n>           FuncionesPropias.inicioSesion(self=driver)\n\ntest_3_CierreDeSesion.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0fe3a5481da884b2695eb161ea3cbf6e\")>, usuario = 'pormero@colegium.com', clave = 'Pop23725'\n\n    def inicioSesion(self, usuario=usuario, clave=clave):\n        print(\"■■■■■-Inicio inicio sesion-■■■■■\")\n>       self.driver.implicitly_wait(20)\nE       AttributeError: 'WebDriver' object has no attribute 'driver'\n\n..\\Funciones\\FuncionesPropias.py:41: AttributeError"}, "attachments": [{"name": "screenshot_error", "source": "53e01a7a-1493-4ec0-afa9-26e36e7e0c1c-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "0efe0665-22f4-4582-b4e3-4993ddf71929-attachment.txt", "type": "text/plain"}], "start": 1723753519566, "stop": 1723753519752, "uuid": "2d9cf0a9-a5c1-4de7-924f-e99a3baa76a3", "historyId": "16640ab113bed520f58f5a95743c105e", "testCaseId": "16640ab113bed520f58f5a95743c105e", "fullName": "test_3_CierreDeSesion#test_login_propio", "labels": [{"name": "feature", "value": "Login Tests"}, {"name": "suite", "value": "test_3_CierreDeSesion"}, {"name": "host", "value": "DESKTOP-6A7N4QP"}, {"name": "thread", "value": "27904-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_3_CierreDeSesion"}]}